mobile/src/components/common/text/style.ts

```diff
@@ -0,0 +1,101 @@
+import { StyleSheet } from 'react-native';
+import { AppColor } from '~/common/enums/ui/app-color.enum';
+import { AppFont } from '~/common/enums/ui/app-font.enum';
+
+const styles = StyleSheet.create({
```

Comment:
Remove these variants too, the only thing that should be here is the default font family, font size and color, everything else we will provide from where it is used

Reply:
What values are default?